spi/control-plane/transfer-spi/src/main/java/org/eclipse/edc/connector/transfer/spi/types/protocol/TransferRequestMessage.java

```diff
@@ -31,9 +31,9 @@ public class TransferRequestMessage implements TransferRemoteMessage {
     private String protocol;
     private String id;
     private String contractId;
-    private String assetId;
+    private String assetId; // TODO remove when removing ids module
```

Comment:
deprecation will work better than TODO in cases like these.

Reply:
Added 
```java
@Deprecated(forRemoval = true)
```